src/peliasDocGenerators.js

```diff
@@ -42,21 +43,28 @@ module.exports.createPeliasDocGenerator = function(hierarchy_finder) {
       switch (hierarchy_element.place_type) {
         case 'locality':
           wofDoc.setAdmin( 'locality', hierarchy_element.name);
+          wofDoc.addParent('locality', hierarchy_element.name, hierarchy_element.id.toString());
           break;
         case 'localadmin':
           wofDoc.setAdmin( 'local_admin', hierarchy_element.name);
+          wofDoc.addParent('localadmin', hierarchy_element.name, hierarchy_element.id.toString());
           break;
         case 'county':
           wofDoc.setAdmin( 'admin2', hierarchy_element.name);
+          wofDoc.addParent('county', hierarchy_element.name, hierarchy_element.id.toString());
           break;
         case 'region':
           wofDoc.setAdmin( 'admin1', hierarchy_element.name);
-          if (hierarchy_element.hasOwnProperty('abbreviation')) {
+          if (!_.isUndefined(hierarchy_element.abbreviation)) {
```

Comment:
Assuming valid `heirarchy_element.abbreviation` values don't include `null`, the integer `0`, `false`, etc, I think this can be simplified to just `if (heirarchy_element.abbreviation)`